airbyte/mcp/connector_registry.py

```diff
@@ -161,6 +179,83 @@ def get_connector_info(
     )
 
 
+@mcp_tool(
+    domain="registry",
+    read_only=True,
+    idempotent=True,
+)
+def get_connector_version_history(
+    connector_name: Annotated[
+        str,
+        Field(
+            description="The name of the connector (e.g., 'source-faker', 'destination-postgres')"
+        ),
+    ],
+    limit: Annotated[
```

Comment:
Let's rename 'limit' to num_versions_to_validate - with a default of 5. And let's move the core implementation (including the base model definition) into the registry model so it's also accessible from the PyAirbyte API.